formbricks/formbricks
# fix: icons not aligned center on firefox

## What does this PR do?
Fixes #1185 


![mozilla centered](https://github.com/formbricks/formbricks/assets/37889962/ca3ff153-0ca8-4dcf-9569-a18b4ba022e1)
![icons centered](https://github.com/formbricks/formbricks/assets/37889962/d7a87d3c-5d2f-4344-8410-55ebf01f0244)

## Type of change




## How should this be tested?


- run the main formbricks.com app and visiting the pricing page using mozilla firefox
- visit the pricing page with google chrome and safari to see

## Checklist



### Required


### Appreciated
